apps/experiments/models.py

```diff
@@ -398,13 +402,21 @@ def get_llm_service(self):
         elif self.assistant:
             return self.assistant.llm_provider.get_llm_service()
 
+    def get_participant_data(self, participant: "Participant") -> "ParticipantData":
+        try:
+            return self.participant_data.get(participant=participant).data
+        except ParticipantData.DoesNotExist:
+            return None
+
     def get_absolute_url(self):
         return reverse("experiments:single_experiment_home", args=[self.team.slug, self.id])
 
 
 class Participant(BaseTeamModel):
     identifier = models.CharField(max_length=320, blank=True)  # max email length
     public_id = models.UUIDField(default=uuid.uuid4, unique=True)
+    external_chat_id = models.CharField(null=False)
```

Comment:
How is this different from `identifier`? It seems like they are always assigned the same value. Maybe we can drop one?

Reply:
You are correct, they are the same. I was thinking of dropping one of these in a followup PR, but maybe only after the data migration was run and things have "settled". It feels like I don't want to lose this data just yet. Hmm, on second thought, if we make a DB dump before the migration it should be fine. (to be clear, I have confidence that the data migration will work, I just don't want to make too many changes at once)